### ISSUE TITLE
Remove invalid component's attributes

### ISSUE THREAD

**Issue by fernandojsg** [aframevr/aframe-inspector#2]
If we have an entity with an standard material like:

``` html
<a-entity geometry="primitive: sphere" material="metalness:0.0;roughness:1.0;shader:standard"></a-entity>
```

And we change the shader, some of the parameters won't be available on this new shader (metalness, roughness):

``` html
<a-entity geometry="primitive: sphere" material="metalness:0.0;roughness:1.0;shader:noise"></a-entity>
```

So probably we should delete them, as it will keep giving warnings on the console:

```
core:schema:warn Unknown component property: metalness +0ms
browser.js:112 core:schema:warn Unknown component property: roughness +2ms
```

**Comment by fernandojsg** [comment]
@dmarcos could you take a look at it? As I believe it may be related with the work you did on the copy to clipbard

**Comment by dmarcos** [comment]
This warning is expected since the schema has changed. I'm not super fan of removing any attributes since it might surprise the user.

**Comment by fernandojsg** [comment]
Yes I'm not sure about removing the attributes either, I feel like it's nice as you're just removing attributes not used with the current schema, but on the other hand it's true that you could delete an attribute that the user could need later, like having a `sphere` with `radius=10`, switching to `box` by mistake and then back to `sphere` and the `radius` value is gone...
@ngokevin what do you think? better to close it?

**Comment by ngokevin** [comment]
Yeah, I'd close. Wouldn't mess with user code. Maybe they expect to change schema later or something.

**Comment by fernandojsg** [comment]
Cool, thanks!
